solidity/contracts/ECDSARewards.sol

```diff
@@ -100,6 +116,67 @@ contract ECDSARewards is Rewards {
         )
     {
         factory = BondedECDSAKeepFactory(_factoryAddress);
+        tokenStaking = TokenStaking(_tokenStakingAddress);
+    }
+
+    /// @notice Get the amount of rewards allocated
+    /// for the specified operator's beneficiary in the specified interval.
+    /// @param interval The interval
+    /// @param operator The operator
+    /// @return The amount allocated
+    function getAllocatedRewards(uint256 interval, address operator)
+        external view returns (uint256) {
+        address beneficiary = tokenStaking.beneficiaryOf(operator);
+        return allocatedRewards[beneficiary][interval];
+    }
+
+    /// @notice Get the amount of rewards already withdrawn
+    /// for the specified operator's beneficiary in the specified interval.
+    /// @param interval The interval
+    /// @param operator The operator
+    /// @return The amount already withdrawn
+    function getWithdrawnRewards(uint256 interval, address operator)
+        external view returns (uint256) {
+        address beneficiary = tokenStaking.beneficiaryOf(operator);
+        return withdrawnRewards[beneficiary][interval];
+    }
+
+    /// @notice Get the amount of rewards withdrawable
+    /// for the specified operator's beneficiary in the specified interval.
+    /// @param interval The interval
+    /// @param operator The operator
+    /// @return The amount withdrawable
+    function getWithdrawableRewards(uint256 interval, address operator)
+        external view returns (uint256) {
+        address beneficiary = tokenStaking.beneficiaryOf(operator);
+        uint256 allocated = allocatedRewards[beneficiary][interval];
+        uint256 withdrawn = withdrawnRewards[beneficiary][interval];
+        return allocated.sub(withdrawn);
+    }
+
+    /// @notice Withdraw all available rewards for the given interval.
+    /// The rewards will be paid to the beneficiary of the specified operator.
+    /// @param interval The interval
+    /// @param operator The operator
+    function withdrawRewards(uint256 interval, address operator) external {
+        address beneficiary = tokenStaking.beneficiaryOf(operator);
+
+        uint256 allocatedForBeneficiary
```

Comment:
We may consider renaming this variable to just `allocated` so that the linter is not getting crazy about tabulating the line.

Reply:
Done in https://github.com/keep-network/keep-ecdsa/pull/584/commits/873cb9a177301a3529a317d79127131c37a991de